src/config/config.service.spec.ts

```diff
@@ -0,0 +1,251 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/*
+https://docs.nestjs.com/fundamentals/testing#unit-testing
+*/
+
+import { Test } from '@nestjs/testing';
+import { describe, it, expect, beforeAll, jest } from '@jest/globals';
+import { ConfigModule } from '@nestjs/config';
+import { ConfigService } from './config.service';
+import { configModuleOptions } from './env.config';
+
+const setupConfigService = async (envObj: any): Promise<ConfigService> => {
+  jest.resetModules();
+  Object.keys(process.env).forEach((key) => {
```

Comment:
question: why are we removing these values?

Reply:
It just resets env variable for some different tests
spec.ts are test files